process/interface.go

```diff
@@ -412,6 +412,7 @@ type RequestHandler interface {
 	RequestRewardTransactions(destShardID uint32, txHashes [][]byte)
 	RequestMiniBlock(shardId uint32, miniblockHash []byte)
 	RequestHeader(shardId uint32, hash []byte)
+	RequestTrieNode(shardId uint32, hash []byte)
```

Comment:
is it only one node that is requested?
This would request a package of nodes starting from node with given hash.

Maybe rename to RequestTrieNodes?